mypy/server/deps.py

```diff
@@ -199,6 +199,8 @@ def visit_class_def(self, o: ClassDef) -> None:
             self.add_type_dependencies(base, target=target)
         if o.info.tuple_type:
             self.add_type_dependencies(o.info.tuple_type, target=make_trigger(target))
+        if o.info.typeddict_type:
```

Comment:
I am mostly in favor of "unification" of ``TypedDict`` and ``NamedTuple``, but why can't you just use `o.info.replaced.typeddict_type` here? (Then you wouldn't need to change `semanal.py`.)

Reply:
`replaced` doesn't get serialized, so the right thing to do seemed to be following `NamedTuple` and actually doing the replacement.